### PR TITLE
added update [standby] on /status through ws #504

### PR DESCRIPTION
**Pull request by sarco3t**
#504

### CODE REVIEW

File: config/initializers/warden_hooks.rb [hunk 1]
```diff
@@ -0,0 +1,13 @@
+# frozen_string_literal: true
+
+Warden::Manager.after_set_user do |user, auth, opts|
+  scope = opts[:scope]
+  auth.cookies.signed["#{scope}.id"] = user.id
+  auth.cookies.signed["#{scope}.expires_at"] = 30.minutes.from_now
```
Comment (ArtOfCode-) [inline config/initializers/warden_hooks.rb]: Question: do we need a user.expires_at cookie? What does it do for us?

Comment (sarco3t) [reply config/initializers/warden_hooks.rb]: I'm not sure, that app need expires_at, but in general cases it is needed.

Comment (ArtOfCode-) [reply config/initializers/warden_hooks.rb]: _shrug_ it probably won't cause problems, so I'm gonna merge and we'll see how it goes.

**Comment by ArtOfCode-** [comment]
Thanks :)

**Pull request merged**